plugins/org.eclipse.sirius.diagram.ui/src-diag/org/eclipse/sirius/diagram/ui/tools/api/format/SiriusStyleApplicator.java

```diff
@@ -177,13 +182,18 @@ private EAttribute getCorrespondingEAttribute(EAttribute attribute, Customizable
             EStructuralFeature structuralFeature = style.eClass().getEStructuralFeature(attribute.getName());
             if (structuralFeature instanceof EAttribute) {
                 result = (EAttribute) structuralFeature;
-            } else if ("color".equals(attribute.getName())) { //$NON-NLS-1$
+            } else if (COLOR.equals(attribute.getName())) {
                 structuralFeature = style.eClass().getEStructuralFeature("backgroundColor"); //$NON-NLS-1$
                 if (structuralFeature instanceof EAttribute) {
                     result = (EAttribute) structuralFeature;
                 }
             } else if ("backgroundColor".equals(attribute.getName())) { //$NON-NLS-1$
```

Comment:
Use "OR" to avoid code duplication `if ("backgroundColor".equals(attribute.getName()) || "foregroundColor".equals(attribute.getName()))`